interface/main/messages/save.php

```diff
@@ -32,10 +32,11 @@
         $data['value']  = $frow['fname']." ".$frow['lname'];
         $data['pid']    = $frow['pid'];
         $data['mobile'] = $frow['phone_cell'];
+        $data['allow']  = $frow['hipaa_allowsms'];
         $results[]      = $data;
     }
     //echo $query. " -- ".$param;
-    echo json_encode($results);
+    echo json_encode( $results );
```

Comment:
Note this formatting breaks PSR2 (shouldn't have those spaces).
(this isn't really something to be worried about since I fixup the PSR2 before bringing it into the codebase in the future)